packages/anu-vue/src/components/base-input/ABaseInput.tsx

```diff
@@ -72,12 +73,12 @@ export const ABaseInput = defineComponent({
 
                     {/* 👉 Slot: Default */}
                     {slots.default?.({
+                      ...ctx.attrs,
```

Comment:
As we already extracting the `class` from `attrs`, we should not add the same classes to the default slot as well

Reply:
If we add only `inputAttrs`, obtained by destructuring using a rest property, we lose reactivity again. Classes will not be added, in my opinion, because they will be overridden by object assign.

```js
{slots.default?.({
  ...ctx.attrs,
  class: [ /* input child classes */ ],
  id: elementId,
})}
```
An alternative solution would be to use the `toRefs()` function to get the reactive `inputAttrs` without classes. However, I'm not sure if this can be used in combination with the rest property.
```js
const { class: rootClasses, ...inputAttrs } = toRefs(attrs)
```
This is my first contribution to the opensource project. I hope I don't break more things than I fix.

Reply:
Let me check tomorrow, what we can do here.

I will let you know, Thanks.

Good night 🌃